src/get-started/npm-script.md

```diff
@@ -5,4 +5,48 @@ title: Get Started > NPM Script
 
 # NPM Script
 
-Need to write up.
+Use npm and `workbox-build` to build a precaching service worker. Simply install the module then cut and paste the code sample.
+
+1. [Install Node.js](https://nodejs.org/en/).
+1. Install the module with NPM.
+
+    ```
+    npm install --save-dev sw-build
+    ```
+
+1. Add the following to your `package.json`:
+
+    ```
+	"scripts": {
+	    "createsw": "node createsw.js"
```

Comment:
This should show taking an existing `build` process and extending it by chaining a call to `workbox-cli` at the end. Like

```
"scripts": {
  "build": "existing-build-step && workbox-cli"
}
```